shared/gh/partials/empty-timetable.html

```diff
@@ -3,7 +3,9 @@
     <i class="fa fa-share-square-o"></i>
 </div>
 <p class="gh-empty-main">Unfortunately this is not in Timetable yet</p>
-<p class="gh-empty-description">Click below to access it, and ask you departmental administrator about making it available via Timetable</p>
 <% if (data.metadata.external) { %>
+    <p class="gh-empty-description">Click below to access it, and ask your departmental administrator about making it available via Timetable</p>
     <a href="<%- data.metadata.external %>" target="_blank" id="gh-empty-access" class="btn btn-default">Access timetable</a>
```

Comment:
We'll need to be a bit more clever about how to add the link here. Links without `http://` wonn't link properly but look like `http://2014.timetable.cam.ac.uk/www.google.co.uk`